package.json

```diff
@@ -24,27 +24,29 @@
     "test:html-coverage": "BABEL_ENV=test nyc --reporter=html npm run test:unit",
     "test:unit": "BABEL_ENV=test jest test/unit/** --verbose",
     "test:unit:watch": "BABEL_ENV=test jest test/unit/** --verbose --watch",
-    "config:restore": "test/integration/scripts/post-test.sh",
-    "test:integration": "npm run test:integration:setup && npm run test:integration:nosetup && npm run config:restore",
-    "test:integration:ci": "npm run test:integration:setup && npm run test:integration:nosetup:ci && npm run config:restore",
-    "test:integration:fresh": "npm run test:integration:setup && npm run test:integration:update && npm run config:restore",
-    "test:integration:setup": "test/integration/scripts/pre-test.sh",
-    "test:integration:nosetup": "BABEL_ENV=test jest test/integration/cmds/** --verbose",
-    "test:integration:nosetup:ci": "BABEL_ENV=test jest test/integration/cmds/** --verbose --runInBand",
-    "test:integration:update": "BABEL_ENV=test jest test/integration/cmds/** --verbose --updateSnapshot",
-    "test:integration:watch": "BABEL_ENV=test jest test/integration/cmds/** --verbose --watch",
+    "pretest:integration": "node test/integration/scripts/pre-test.js",
+    "pretest:integration:ci": "npm run pretest:integration",
```

Comment:
the duplication in these scripts is not ideal, but this is the best I could come up with so that we could pass the `--runInBand` flag to the integration tests on CI only. running tests without that flag takes less than half the time, so I don't want to enforce it for local tests if we don't have to.

any ideas for improving this are welcome!

Reply:
👍

Reply:
improving the workflow is an ongoing thing as long as we update the docs everytime, I think it should be fine